src/Symfony/Component/Security/Core/Authorization/ExpressionLanguage.php

```diff
@@ -13,23 +13,27 @@
 
 use Symfony\Component\ExpressionLanguage\ExpressionLanguage as BaseExpressionLanguage;
 
-/**
- * Adds some function to the default ExpressionLanguage.
- *
- * @author Fabien Potencier <fabien@symfony.com>
- *
- * @see ExpressionLanguageProvider
- */
-class ExpressionLanguage extends BaseExpressionLanguage
-{
+if (!class_exists('Symfony\Component\ExpressionLanguage\ExpressionLanguage')) {
+    throw new \RuntimeException('The ExpressionLanguage class requires the "ExpressionLanguage" component. Install "symfony/expression-language" to use it.');
```

Comment:
`The "%s" class`, with FQCN and double quotes around, with ExpressionLanguage::class given

Reply:
It's not the case in others similar messages (in the Serializer component for example). I guess we should harmonize them ?

Reply:
Actually not: we don't harmonize such CS things because they create merge conflicts when merging lower branches into upper ones.